crates/valence_boss_bar/src/lib.rs

```diff
@@ -14,16 +14,115 @@ impl Plugin for BossBarPlugin {
 
     fn build(&self, app: &mut bevy_app::App) {
         app
-        .add_system(remove_despawned_boss_bars_from_viewers);
+        .add_system(remove_despawned_boss_bars_from_viewers)
+        .add_system(handle_boss_bar_title_update)
+        .add_system(handle_boss_bar_health_update)
+        .add_system(handle_boss_bar_style_update)
+        .add_system(handle_boss_bar_flags_update)
+        .add_system(handle_boss_bar_viewers_update);
     }
 
 }
 
+fn handle_boss_bar_title_update(mut boss_bars: Query<(&UniqueId, &BossBarTitle, &mut BossBarViewers), Changed<BossBarTitle>>, mut clients: Query<&mut Client>) {
```

Comment:
I'm sure that's not the right way to make a query but I could not find how to query the bundle itself.

Reply:
I don't think there's a way to query bundles